templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -136,3 +136,24 @@ a space before your phrase or submit it as a quote block by following
 the code syntax highlighting format.
 
 ![Quotes](/static/images/help/quotes-screenshot.png)
+
+## TeX math
+
+You can display TeX mathematical expressions in Zulip.
+[TeX](http://www.tug.org/interest.html#doc), amongst other things, allow you to
+easily write formulas through commands. Zulip's implementation of this
+typesetting is [KaTeX](https://github.com/Khan/KaTeX), and you can see the full
+set of  compatible commands
+[here](https://github.com/Khan/KaTeX/wiki/Function-Support-in-KaTeX).
+
+For inline content, surround
```

Comment:
"Surround elements in valid TeX syntax with `$$two dollar signs$$` to display it as inline content."